File: web-dev/helsinki-mooc/python/ex69.py
def reverse_text(text):
    return text[::-1]

def pallindrome(text):
    if text == reverse_text(text):
        return True
    else:
        return False

user_text = input('Type a text: ')

if pallindrome(user_text) == True:
    print('The word \'', user_text, '\' is pallindrome')
else:
    print('The word is not pallindrome')

File: programming/helsinki-mooc/java/ex69.java
import java.util.Scanner;
import java.util.ArrayList;

public class ex69 {

  public static boolean is_pallindrome(String word) {
    String reversed_word = "";
    for (int i = word.length()-1; i > 0; i--) {
      reversed_word += word.charAt(i);
    }
    if (reversed_word == word) {
      return true;
    } else {
      return false;
    }
  }

  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);

    System.out.println("Type a word: ");
    String word = input.nextLine();

    if (is_pallindrome(word)) {
      System.out.println("The word is pallindrome !");
    } else {
      System.out.println("The word is NOT pallindrome !");
    }
  }

}
